Project.toml

```diff
@@ -6,6 +6,7 @@ version = "13.5.1"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+DynamicPolynomials = "7c1d4256-1411-5781-91ec-d7bc3513ac07"
```

Comment:
Required to handle HomotopyCotninuation for systems featuring equations.

Reply:
Shouldn’t this be an extra?

Reply:
Because it is only relevant to an extension? If I understand the question correct (unfortunately) no, I asked about it on the Julai slack earlier:
![image](https://github.com/SciML/Catalyst.jl/assets/18099310/1c5f9a88-67f6-45a1-96d9-99ffb30672b6)

Reply:
Actually, it should already be being pulled installed as a Symbolics dependency anyways it seems.